tests/unittest/test_models.py

```diff
@@ -51,3 +52,51 @@ def test_text_models():
             model.collect_params().initialize()
         output, state = model(mx.nd.arange(330).reshape(33, 10))
         output.wait_to_read()
+
+def test_cache_models():
+    cache_language_models = ['awd_lstm_lm_1150', 'awd_lstm_lm_600', 'standard_lstm_lm_200',
+                   'standard_lstm_lm_650', 'standard_lstm_lm_1500']
+    datasets = ['wikitext-2']
+    for name in cache_language_models:
+        for dataset_name in datasets:
+            cache_cell = get_cache_model(name, dataset_name, window=1, theta=0.6,
+                                         lambdas=0.2, root='tests/data/model/')
+            outs, word_history, cache_history, hidden = \
+                cache_cell(mx.nd.arange(10).reshape(10, 1), mx.nd.arange(10).reshape(10, 1), None, None)
+            print(cache_cell)
+            print("outs:")
+            print(outs)
+            print("word_history:")
+            print(word_history)
+            print("cache_history:")
+            print(cache_history)
+
+
+def test_get_cache_model_noncache_model():
```

Comment:
thanks for adding the test. could you add a test case for saving from cache model and loading from regular model too?